packages/neuron-wallet/.env

```diff
@@ -120,3 +120,10 @@ MULTISIG_CODE_HASH=0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc963
 CKB_NODE_ASSUME_VALID_TARGET='0x6dd077b407d019a0bce0cbad8c34e69a524ae4b2599b9feda2c7491f3559d32c'
 CKB_NODE_ASSUME_VALID_TARGET_BLOCK_NUMBER=13007704
 CKB_NODE_DATA_SIZE=56
+
+# openssl rsa -in key.pem -outform PEM -pubout -out public.pem
+# LOG_ENCRYPTION_PUBLIC_KEY="-----BEGIN PUBLIC KEY-----
+# 256-bit RSA KEY IN PEM FORMAT
+# -----END PUBLIC KEY-----"
+# TODO
+LOG_ENCRYPTION_PUBLIC_KEY="-----BEGIN RSA PUBLIC KEY-----\n base64 RSA public key here \n-----END RSA PUBLIC KEY-----"
```

Comment:
The environment variable has a higher priority than `.env`. I left the field in `.env` config mainly as a hint that this field needs to be configured. We can leave it empty here.

Reply:
It's better to leave it, or mention it in a file that won't be tracked by git by default. Otherwise, users may set their own key in the file and publish it on github

Reply:
I don't think the reason makes sense. It's much more profitable to just change those script configs above to your own deployed malicious contracts than it is to steal the error logs

Reply:
> I don't think the reason makes sense. It's much more profitable to just change those script configs above to your own deployed malicious contracts than it is to steal the error logs

Removing `public key` from the `.env` is not for defending malicious code, it's for careless developers. There are always developers uploading their own key to GitHub.